example.config.yml

```diff
@@ -117,6 +123,7 @@ firewall_allowed_tcp_ports:
   - "8080"
   - "8443"
   - "8983"
+  - "35729"
```

Comment:
Let's leave the default set of ports as-is, and just make sure the documentation page for Livereload mentions adding the specified port to the firewall allowed TCP ports.